frontend/src/employee-frontend/components/ChildInformation.tsx

```diff
@@ -95,6 +97,7 @@ const layouts: Layouts<typeof components> = {
     { component: 'backup-care', open: false },
     { component: 'service-need', open: false },
     { component: 'daily-service-times', open: false },
+    { component: 'vasuAndLeops', open: false },
```

Comment:
even though this is just for admin, maybe we should introduce a new feature flag into config.ts and set it to false in prod and maybe staging too?

Reply:
PS: Panu has feature flag in https://github.com/espoon-voltti/evaka/pull/1013 so sync with that

Reply:
Feature flag implemented.